api/models.py

```diff
@@ -184,6 +184,20 @@ class Meta:
         db_table = "api_quiz_session_student"
 
 
+class QuizSessionLog(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    quiz_session_code = models.CharField(max_length=6, unique=True)
+    quiz_session = models.ForeignKey(QuizSession, on_delete=models.CASCADE)
+    quiz_session_student = models.ForeignKey(QuizSessionStudent, on_delete=models.CASCADE)
+    question_multiple_choice = models.ForeignKey(QuestionMultipleChoice, on_delete=models.CASCADE)
```

Comment:
Let's allow null for `question_multiple_choice`, just in case that value was missing.